tests/utils/verification/volumeproperties.go

```diff
@@ -24,22 +24,22 @@ import (
 	sshutil "github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
 )
 
-var ADMIN_CLI_LS = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls "
-var DOCKER_CLI_INSPC = "docker volume inspect "
+var adminCliLs = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls "
```

Comment:
I agree with Shahzeb here. Actually we shouldn't rename these commands to camel case, which makes the code harder to read. It's almost a standard convention to define CLI commands in capital letters.